vite.config.cjs

```diff
@@ -1,30 +1,35 @@
+const glslify = require("rollup-plugin-glslify");
+import { defineConfig } from "vite";
 
-const glslify = require('rollup-plugin-glslify');
 //import worker from 'rollup-plugin-web-worker-loader';
 
 //import { fileURLToPath } from 'url';
 
 //const __filename = fileURLToPath(import.meta.url);
 //const __dirname = path.dirname(__filename);
 
-export default {
+export default defineConfig(({ mode }) => ({
+  server: {
+    open:
+      mode === "simple" ? "/index-simplest-way-to-start.html" : "/index.html",
+  },
   build: {
```

Comment:
^ this is the change.

I just had to use defineConfig in order to read off mode.